src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -205,16 +204,14 @@ string[] additionalModules
                     logger);
 
             Runspace initialRunspace = PowerShellContextService.CreateRunspace(psHost);
+            var profilePaths = new ProfilePaths(hostDetails.ProfileId, hostDetails.AllUsersProfilePath, hostDetails.CurrentUserProfilePath);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename 'profilePaths' which hides the field with the same name.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4579423)